prow/config/config_test.go

```diff
@@ -90,6 +91,15 @@ func TestDefaultJobBase(t *testing.T) {
 				j.Namespace = &p
 			},
 		},
+		{
+			name: "empty cluster becomes DefaultClusterAlias",
```

Comment:
Do we have a test for `error_on_eviction` with non-kubernetes agent?

Reply:
We do now.